static/output.js

```diff
@@ -77,6 +81,14 @@ define(function (require) {
         if (id === this.compilerId) this.fontScale.setScale(scale);
     };
 
+    Output.prototype.programOutput = function (msg, color) {
+        var elem = $('<div></div>').appendTo(this.contentRoot);
```

Comment:
I think you can chain the `.text` and `.css` methods with this line. Keep them in separate lines, but there's no need to `elem.` before each one